flight/Modules/Actuator/actuator.c

```diff
@@ -572,193 +724,27 @@ static float channel_failsafe_value(int idx)
  */
 static void set_failsafe()
 {
-	float Channel[ACTUATORCOMMAND_CHANNEL_NUMELEM];
-
-	ActuatorCommandChannelGet(Channel);
+	float Channel[ACTUATORCOMMAND_CHANNEL_NUMELEM] = {0};
```

Comment:
Can you please make this lowercase? (you touched every line except 1 already :P)